src/utils/onResize/onResize.js

```diff
@@ -0,0 +1,8 @@
+const onResize = setState => ($container, $content) => {
```

Comment:
Again `onResize` does not tell me anything about what's this function is doing. Also just returning the object and then setting the state where the function is invoked will be more clean.

Reply:
Names suggestions: `isContentScrollable`, `getIsScrollableState`, `getScrollableDimensions`